app/src/main/kotlin/com/intive/picover/parties/repository/MockedPartiesRepository.kt

```diff
@@ -1,21 +1,27 @@
 package com.intive.picover.parties.repository
 
-import com.intive.picover.parties.model.Party
+import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.ktx.snapshots
+import com.intive.picover.parties.model.PartyRemote
 import javax.inject.Inject
-import javax.inject.Singleton
+import kotlinx.coroutines.flow.map
 
-@Singleton
-class MockedPartiesRepository @Inject constructor() {
+class MockedPartiesRepository @Inject constructor(
+	firebaseDatabase: FirebaseDatabase,
+) {
 
-	private val parties = listOf(
-		Party(id = 1, title = "title1", description = "description1"),
-		Party(id = 2, title = "title2", description = "description2"),
-		Party(id = 3, title = "title3", description = "description3"),
-		Party(id = 4, title = "title4", description = "description4"),
-		Party(id = 5, title = "title5", description = "description5"),
-	)
+	private val partiesReference =
+		firebaseDatabase.getReference("Parties")
 
-	fun getParties() = parties
+	fun parties() =
+		partiesReference.snapshots.map { snapshot ->
+			snapshot.children.map {
+				it.getValue(PartyRemote::class.java)!!
```

Comment:
You used `snapshot` above and here `it`, IMO to save consistency you can name it correctly to avoid using `it` in this case

Reply:
ok, use childSnapshot :D